scripts/tsconfig.json

```diff
@@ -26,9 +26,9 @@
 		// "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
 		/* Modules */
-		"module": "CommonJS" /* Specify what module code is generated. */,
+		"module": "ES2022" /* Specify what module code is generated. */,
```

Comment:
I wonder if we need to make the same changes in `tsconfig.json` in the root. It's a bit weird that they differ. Honestly, I don't fully understand why we need two files at all. 🤷‍♂️

Reply:
Basically, my understanding is running the scripts to build the project is very different from the compiler options to actually build and use the project. For example, you have to compile the project for use on NPM rather than simply run the code.

Reply:
I still don't understand why they should differ in our case. 🫣 But I think it's orthogonal to this PR anyway. 🙂